package.json

```diff
@@ -6,16 +6,19 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start -p $PORT",
+    "export": "next export",
```

Comment:
Most of these changes are in the most recent commit on main branch, so just merge those changes into this branch